www/js/diary/infinite_scroll_list.js

```diff
@@ -32,7 +33,7 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
                                     ionicDatePicker,
                                     $timeout,
                                     leafletData, Timeline, CommonGraph, DiaryHelper,
-                                    SurveyOptions,
+                                    SurveyOptions, NotificationScheduler,
```

Comment:
For the record, I assume that you are including this into the label screen instead of the intro because you can be guaranteed that the user has logged in and you can update the profile.

Is that right?

Reply:
Correct